src/cloud_manager.ts

```diff
@@ -13,9 +15,11 @@ import logger from './logger';
 
 export default class CloudManager {
     private cloud = 'aws';
+    private oracleCloudManager: OracleCloudManager;
 
-    constructor(options: CloudManagerOptions) {
+    constructor(options: CloudManagerOptions, oracleOptions: OracleCloudManagerOptions) {
```

Comment:
The CloudManagerOptions param is meant to contain all input params for the class, so we can add oracleOptions as a field on the CloudMangerOptions rather than passing it as a separate parameter.